### PR TITLE
WC close wallet sheet on wallet change

### PR DESCRIPTION
**Pull request by estebanmino**
In wallet connect connection sheet, close change wallet sheet on wallet change

https://recordit.co/AzLno3K3W1

### CODE REVIEW

File: src/screens/WalletConnectApprovalSheet.js [hunk 1]
```diff
@@ -223,11 +223,13 @@ export default function WalletConnectApprovalSheet() {
   const handlePressChangeWallet = useCallback(() => {
     Navigation.handleAction(Routes.CHANGE_WALLET_SHEET, {
       currentAccountAddress: approvalAccount.address,
-      onChangeWallet: (address, wallet) =>
-        setApprovalAccount({ address, wallet }),
+      onChangeWallet: (address, wallet) => {
+        goBack();
+        setApprovalAccount({ address, wallet });
```
Comment (skylarbarrera) [inline src/screens/WalletConnectApprovalSheet.js:228]: i feel like it would be better for these lines to be flipped in order, goBack after setAccount

**Review by skylarbarrera — APPROVED** [review]
very nice catch, lgtm 💯

File: src/screens/WalletConnectApprovalSheet.js (hunk 1 above)
Comment (estebanmino) [reply src/screens/WalletConnectApprovalSheet.js:228]: yeah I thought the same but we're not going back from this component, we're just closing the sheet on top of it. I can do it tho

Comment (estebanmino) [reply src/screens/WalletConnectApprovalSheet.js:227]: done 👌

Comment (skylarbarrera) [reply src/screens/WalletConnectApprovalSheet.js:227]: thank u ser, merge whenever 🙏

**Pull request merged**